cli/src/cli_util.rs

```diff
@@ -2001,6 +2015,32 @@ pub fn print_trackable_remote_branches(ui: &Ui, view: &View) -> io::Result<()> {
     Ok(())
 }
 
+pub fn check_dropped_signature(
+    ui: &Ui,
+    old_commit: &Commit,
+    rewritten: &Commit,
+) -> Result<(), std::io::Error> {
```

Comment:
nit: maybe this can be inlined? It's not obvious at the call site whether the arguments are `(old, new)` or `(new, old)`.